src/test/javascript/spec/component/shared/vertical-progress-bar.component.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { VerticalProgressBarComponent } from 'app/shared/vertical-progress-bar/vertical-progress-bar.component';
+import { runOnPushChangeDetection } from '../../helpers/on-push-change-detection.helper';
+
+describe('VerticalProgressBarComponent', () => {
+    let component: VerticalProgressBarComponent;
+    let fixture: ComponentFixture<VerticalProgressBarComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [VerticalProgressBarComponent],
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(VerticalProgressBarComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
```

Comment:
Does this test serve any purpose? The other test will fail, too, if the component can't be created. :)

Reply:
Removed in 1b0345999922a4ab85d31a1f707a15344fadab9e